src/index.js

```diff
@@ -34,6 +34,26 @@ export function toggleCreator(context) {
   }
 }
 
+export function handleSumbit(func){
+  return (e) => {
+    e.preventDefault();
+
+    const form = e.target;
+    let values = {};
+
+    let children = [];
+
+    children.push(form.querySelectorAll('input, select, textarea'));
```

Comment:
Elements is the exact same list

https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/elements

I wasn't able to pull out buttons in a test right now, hmm.